mypyc/test-data/fixtures/ir.py

```diff
@@ -238,6 +238,7 @@ class StopIteration(Exception):
 
 def any(i: Iterable[T]) -> bool: pass
 def all(i: Iterable[T]) -> bool: pass
+def sum(i: Iterable[T]) -> int: pass
```

Comment:
Ah, this will need to take a start argument. It looks like that is the cause of a bunch of the test failures.